webapp/channels/src/components/file_info_preview/file_info_preview.tsx

```diff
@@ -13,58 +13,61 @@ type Props = {
     canDownloadFiles: boolean;
 };
 
-export default class FileInfoPreview extends React.PureComponent<Props> {
-    render() {
-        const fileInfo = this.props.fileInfo;
-        const fileUrl = this.props.fileUrl;
+const FileInfoPreview = ({fileInfo, fileUrl, canDownloadFiles}: Props) => {
```

Comment:
Nit: Can you put every prop in a different line? As in:
```suggestion
const FileInfoPreview = ({
    fileInfo,
    fileUrl,
    canDownloadFiles,
}: Props) => {
```

This way, if we ever make any change to this component, is way more visible what props are removed / added.

Reply:
Got it, Thanks